src/plugins/intel_npu/src/plugin/npuw/partitioning/partitioning.cpp

```diff
@@ -1946,6 +1958,17 @@ void Partitioner::optimize(const std::string& func_name) {
                 funcall.get()._host_gather = ov::npuw::Subgraph::Gather{gather_dst_id, gather_src_id, gather_idx_id};
             }
         }
+
+        // FIXME: workaround
+        // Set lazy unpack indexes not to be unpacked in DCOFF
+        for (auto&& fref : func_group.refs) {
+            auto& funcall = fref.get();
+            for (std::size_t idx = 0; idx < funcall._is_lazy_unpack.size(); ++idx) {
```

Comment:
iterating over a `vector<bool>` is meh..

Reply:
Welp, I was forced to use `vector<bool>` due to concat shifting indexes